lib/tasks/wos.rake

```diff
@@ -1,4 +1,11 @@
 namespace :wos do
+  desc 'Harvest from Web of Science, for all authors'
+  task harvest_authors: :environment do
+    Author.where(active_in_cap: true, cap_import_enabled: true)
+          .find_in_batches(batch_size: 200)
+          .each { |authors| WOS.harvester.harvest(authors) }
```

Comment:
This just copies the code from the base class.  That seems unnecessary.

Reply:
Yes.  Initially, when I tried to inherit from the base class, this task was just `WOS.harvester.harvest_all`.  It did not work because some specs went recursive.  I don't understand why and don't have time to resolve it, so this solution was adopted to get an expedient result that works.  Let's spin off a new issue to work on the WOS-harvester to use the base class (I don't have to time to replicate the problem and resolve it in PR).